swagger2.cabal

```diff
@@ -93,11 +97,11 @@ test-suite spec
     Data.Swagger.Schema.ValidationSpec
   default-language: Haskell2010
 
-test-suite doctest
-  build-depends:    base, doctest, Glob
+test-suite doctests
+  build-depends:    base, doctest, Glob, QuickCheck
```

Comment:
we have one `prop>`, it requires `QuickCheck` then.

This is a little hack, actually we don't need `QuickCheck` to build `doctests.hs`; **but** this dependencies are solved by Cabal, so we can pass them as `--package-id=...` to `doctests` executable, so `doctest` can use / import `QuickCheck` modules.